ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -6306,7 +6389,7 @@ components:
             - `TRAC` (TRACE Original)
             - `BRIT` (BRITC eBL)
 
-            **Conditional:** only applicable when `isElectronic=true` and `transportDocumentTypeCode=BOL`. The property **must** be absent for paper B/Ls (`isElectronic=false`)
+            **Condition:** Only applicable when `isElectronic=true` and `transportDocumentTypeCode=BOL`. The property **must** be absent for paper B/Ls (`isElectronic=false`)
```

Comment:
should we capitalize must?

Reply:
I thought about this as half was capitalized and the rest was not. I then agreed with myself that it looks nicer with capitalization - so I changed it everywhere :-)